content/docs/latest/installing/community-platforms/ovhcloud.md

```diff
@@ -0,0 +1,144 @@
+```
+
+The `coreos-metadata.service` saves metadata variables to `/run/metadata/flatcar`. Systemd units can use them with `EnvironmentFile=/run/metadata/flatcar` in the `[Service]` section when setting `Requires=coreos-metadata.service` and `After=coreos-metadata.service` in the `[Unit]` section.
+
+## Launch machine
+
+From here, there are a few options:
+* boot the instance from the OVH Cloud portal
+* boot the instance from the OpenStack CLI
+
+### From the OVH Cloud portal
+
+The official [documentation][ovhcloud-documentation-create] can be used except for the step 3 "Select an image": you need to go to "Backups" to select your imported image (e.g "flatcar-beta").
+
+You can also add the SSH key to be used in the creation form (it will be fetched and injected into Flatcar by the `coreos-metadata-sshkeys@.service`).
+
+To use Ignition, you need to add the content of the Ignition configuration into the "Post-install script" available at the step 4 "Configure your instance".
+
+### From the OpenStack CLI
+
+Boot the machine with the CLI, referencing the image ID from the import step above, your [Ignition file from Butane][butane-configs], a flavor of your choice and a network of your choice:
+
+```shell
+$ INSTANCE_ID=$(openstack server create \
+  --network "${NETWORK_ID}" \
+  --flavor "${FLAVOR_ID}" \
+  --user-data ./ignition.json \
+  --image "${IMAGE_ID}" \
+  --format json \
+  flatcar-01 | jq -r .id)
+```
+
+Your first Flatcar instance should now be running. At this point, the instance should be visible in the OVH Cloud portal too. The only thing left to do is find the IP address and SSH in.
+
+Example:
+```shell
+$ openstack server show "${INSTANCE_ID}" --format json | jq -r .addresses
+{
+  "Ext-Net": [
+    "57.128.163.211",
+    "2001:41d0:801:1000::10d4"
+  ]
+}
+$ curl 57.128.163.211
+Hello from flatcar-01
+```
+
+Finally SSH into an instance, note that the user is `core`:
+
+Example:
+```shell
+$ ssh core@57.128.163.211
+$ systemctl status nginx
+● nginx.service - NGINX example
+     Loaded: loaded (/etc/systemd/system/nginx.service; enabled; preset: enabled)
+     Active: active (running) since Tue 2024-05-28 14:36:37 UTC; 6min ago
+    Process: 1586 ExecStartPre=/usr/bin/docker rm --force nginx1 (code=exited, status=0/SUCCESS)
+    Process: 1592 ExecStartPre=/usr/bin/bash -c echo 'Hello from ${COREOS_OPENSTACK_HOSTNAME}' > /var/www/index.html (code=exited, status=0/SUCCESS)
+    ...
+```
+
+## Using Flatcar Container Linux
+
+Now that you have a machine booted it is time to play around. Check out the [Flatcar Container Linux Quickstart][quickstart] guide or dig into [more specific topics][doc-index].
+
+[butane-configs]: ../../provisioning/config-transpiler
+[doc-index]: ../../
+[ovhcloud]: https://www.ovhcloud.com/en/
+[ovhcloud-documentation-create]: https://support.us.ovhcloud.com/hc/en-us/articles/360002245164-Creating-and-Connecting-a-Public-Cloud-Instance
```

Comment:
```suggestion
[ovhcloud-documentation-create]: https://help.ovhcloud.com/csm/en-public-cloud-compute-getting-started?id=kb_article_view&sysparm_article=KB0051009
```
As the US documentation is handled separately from the rest, it's best to refer to the worldwide one.